courses-portlet-webapp/src/main/webapp/css/degree-progress.css

```diff
@@ -1,215 +0,0 @@
-.degree-progress .fl-flex-2 {
```

Comment:
@doodelicious -- This CSS file is removed, but I don't see style definitions for the degree-progress portlet in the LESS... am I missing something?  (There are currently 6 portlets in the courses-portlet-webapp project:  courses, myCourses, ClassSchedule, finalGrades, course-catalog, and degree-progress.)